### PR TITLE
ui: Domains, Addresses and Mailboxes pages

### PR DESCRIPTION
**Pull request by andre8244**
- UI for **Domains**, **Addresses** and **Mailboxes** pages
- Fix app metadata
- Update `ns8-ui-lib`

Requires #3

### CODE REVIEW

File: ui/public/i18n/language.json [hunk 1]
```diff
@@ -53,5 +78,142 @@
...
+    "destinations": "Destinations",
+    "visibility": "Visibility",
+    "edit_address": "Edit address",
+    "create_address_address": "Create address {address}",
+    "edit_address_address": "Edit address {address}",
+    "internal_address": "Internal address",
+    "internal": "Internal",
+    "public": "Public",
+    "choose_domain": "Choose domain",
+    "destinations_placeholder": "Choose destinations or input external email addresses",
```
Comment (DavidePrincipi) [inline ui/public/i18n/language.json:144]:
```suggestion
    "destinations_placeholder": "Choose destinations or type an external email addresses",
```

File: ui/public/i18n/language.json [hunk 2]
```diff
@@ -53,5 +78,142 @@
...
+    "description": "Address description",
+    "destinations_required": "Choose or input at least one destination",
```
Comment (DavidePrincipi) [inline ui/public/i18n/language.json:146]:
```suggestion
    "destinations_required": "Choose or type at least one destination",
```

File: ui/public/i18n/language.json [hunk 3]
```diff
@@ -53,5 +78,142 @@
...
+    "col_forward": "Forward addresses",
+    "unlimited": "Unlimited",
+    "none": "None",
+    "value_of_total": "{value} of {total}",
+    "settings": "Settings",
+    "edit_mailbox_mailbox": "Edit mailbox {mailbox}",
+    "forward_messages": "Forward messages",
+    "edit_mailbox": "Edit mailbox",
+    "forward_addresses": "Forward addresses",
+    "destinations_placeholder": "Choose destinations or input external email addresses",
```
Comment (DavidePrincipi) [inline ui/public/i18n/language.json:178]:
```suggestion
    "destinations_placeholder": "Choose destinations or type an external email addresses",
```

**Review by DavidePrincipi — COMMENTED** [review]
Labels: what about using word "type" instead of "input"?

**Review by edospadoni — APPROVED** [review]

**Pull request merged**